pkgs/development/compilers/ante/default.nix

```diff
@@ -0,0 +1,60 @@
+{ fetchFromGitHub
+, lib
+, libffi
+, libxml2
+, llvmPackages_13
+, ncurses
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "ante";
+  version = "unstable-2022-07-24";
+  src = fetchFromGitHub {
+    owner = "jfecher";
+    repo = "ante";
+    rev = "187c284ecd3d8d60881f2f55feef47a051654c71";
+    sha256 = "sha256-drNEUISPgVZSFYjWh+yNMEx3aL+lKRQl8nQAlWRogbY=";
+  };
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+
+    outputHashes = {
+      "inkwell-0.1.0" = "sha256-+ih3SO0n6YmZ/mcf+rLDwPAy/1MEZ/A+tI4pM1pUhvU=";
+    };
+  };
+
+  nativeBuildInputs = [ llvmPackages_13.llvm ];
+  buildInputs = [ libffi libxml2 ncurses ];
+
+  postPatch = ''
+    substituteInPlace tests/golden_tests.rs --replace \
+      'target/debug' "target/$(rustc -vV | sed -n 's|host: ||p')/debug"
+
+    cp ${./Cargo.lock} Cargo.lock
+  '';
+  preBuild =
+    let
+      major = lib.versions.major llvmPackages_13.llvm.version;
+      minor = lib.versions.minor llvmPackages_13.llvm.version;
+      llvm-sys-ver = "${major}${builtins.substring 0 1 minor}";
+    in
+    ''
+      export LLVM_SYS_${llvm-sys-ver}_PREFIX=${llvmPackages_13.llvm.dev}
+      export ANTE_STDLIB_DIR=$out/lib;
+      export RUST_BACKTRACE=1
```

Comment:
Do we need this?

Reply:
No, I forgot to remove it after debugging build errors.